real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/ui/AngleGraph.java

```diff
@@ -51,25 +34,36 @@ public void updateGraph(int angle) {
 	private class Graph extends JPanel {
 		private static final long serialVersionUID = 6015150544448011207L;
 
-		private static final int WIDTH = 200;
-		private static final int HEIGHT = 200;
+		private int WIDTH = 200;
+		private int HEIGHT = 200;
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [The field name indicates a constant but its modifiers do not](https://www.codacy.com/app/Mr-Trevor-Decker/RoboBuggy/file/1746561337/issues/source?bid=2938665&fileBranchId=3059924#l38)